### PR TITLE
Feature: Configuration wizard

### PR DESCRIPTION
**Pull request by David-DB88**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
created wizard 

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [x] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by armenuikafka — APPROVED** [review]
Common tasks are reported separately.

File: documentation/compose/kafka-ssl-components.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ services:
     image: provectuslabs/kafka-ui:latest
     ports:
       - 8080:8080
+      - 5005:5005
```
Comment (Haarolean) [inline documentation/compose/kafka-ssl-components.yaml:9]:
```suggestion
```

File: documentation/compose/kafka-ui-jmx-secured.yml [hunk 2]
```diff
@@ -10,24 +10,22 @@ services:
...
-        -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005
-        -Djavax.net.ssl.trustStore=/jmx/clienttruststore
-        -Djavax.net.ssl.trustStorePassword=12345678
-        -Djavax.net.ssl.keyStore=/jmx/clientkeystore
-        -Djavax.net.ssl.keyStorePassword=12345678
+      KAFKA_CLUSTERS_0_METRICS_KEYSTORE_LOCATION: /jmx/clientkeystore
+      KAFKA_CLUSTERS_0_METRICS_KEYSTORE_PASSWORD: '12345678'
+      KAFKA_CLUSTERS_0_SSL_TRUSTSTORE_LOCATION: /jmx/clienttruststore
+      KAFKA_CLUSTERS_0_SSL_TRUSTSTORE_PASSWORD: '12345678'
+#      JAVA_OPTS: >-
```
Comment (Haarolean) [inline documentation/compose/kafka-ui-jmx-secured.yml:26]: remove

File: documentation/compose/kafka-ui-serdes.yaml [hunk 3]
```diff
@@ -14,13 +14,16 @@ services:
             kafka.clusters.0.name: SerdeExampleCluster
             kafka.clusters.0.bootstrapServers: kafka0:29092
             kafka.clusters.0.schemaRegistry: http://schemaregistry0:8085
-            # optional auth and ssl properties for SR
+
+            # optional SSL settings for cluster (will be used by SchemaRegistry serde, if set)
+            #kafka.clusters.0.ssl.keystoreLocation: /kafka.keystore.jks
```
Comment (Haarolean) [inline documentation/compose/kafka-ui-serdes.yaml:19]: @iliax please update the docs in docs branch

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/ClustersStorage.java [hunk 4]
```diff
@@ -5,9 +5,11 @@
 import com.provectus.kafka.ui.model.KafkaCluster;
 import java.util.Collection;
 import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Component;
 
 @Component
+@Slf4j
```
Comment (Haarolean) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/ClustersStorage.java:12]: unused?

File: kafka-ui-api/src/main/resources/application.yml [hunk 5]
```diff
@@ -16,7 +16,5 @@ logging:
   level:
     root: INFO
     com.provectus: DEBUG
-    #org.springframework.http.codec.json.Jackson2JsonEncoder: DEBUG
```
Comment (Haarolean) [inline kafka-ui-api/src/main/resources/application.yml:19]: let's keep this in app-local.yml

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/rbac/Permission.java [hunk 6]
```diff
@@ -25,22 +26,28 @@
...
-    this.value = Pattern.compile(value);
+  @SuppressWarnings("unused")
+  public void setValue(@Nullable String value) {
+    this.value = value;
   }
 
   @SuppressWarnings("unused")
   public void setActions(List<String> actions) {
+    if (actions != null) {
+      actions = actions.stream().map(String::toUpperCase).toList();
```
Comment (Haarolean) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/rbac/Permission.java:49]: looks redundant due to `.map(String::toUpperCase)` in AccessControlService#isAccessible

**Review by Haarolean — APPROVED** [review]

**Review by Haarolean — APPROVED** [review]

**Review by workshur — APPROVED** [review]

**Pull request merged**